model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -525,6 +525,11 @@ message TestStreamPayload {
   // used to retrieve events.
   ApiServiceDescriptor endpoint = 3;
 
+  // (Optional) The PCollection tags this TestStream will be outputting to. If
+  // empty, this will assume it will be outputting to the single main
+  // output PCollection.
+  repeated string output_tags = 4;
```

Comment:
Don't think this is worth duplicating information already present on existing test stream events.

Reply:
Without this field of "output_tags" it is impossible to recreate a TestStream from the TestStreamPayload without also giving the events.

Reply:
This information duplicates the "output" map keys on the PTransform that contains the TestStreamPayload.

Reply:
Cool, got rid the of the output_tags.